645b215a29f36ff7921f0180d9477a8a15df88b4
apollo-common/src/test/java/com/ctrip/framework/apollo/common/utils/WebUtilsTest.java
--- 
+++ 
@@ -1,45 +1,45 @@
 /*
- * Copyright 2022 Apollo Authors
+ * Copyright 2023 Apollo Authors
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
  *
  * http://www.apache.org/licenses/LICENSE-2.0
  *
  * Unless required by applicable law or agreed to in writing, software
  * distributed under the License is distributed on an "AS IS" BASIS,
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
  *
  */
 package com.ctrip.framework.apollo.common.utils;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.mockito.junit.MockitoJUnitRunner;
 import org.springframework.mock.web.MockHttpServletRequest;
 
 /**
  * @author kl (http://kailing.pub)
  * @since 2022/8/12
  */
 @RunWith(MockitoJUnitRunner.class)
 public class WebUtilsTest {
 
   @Test
   public void testTryToGetClientIp() {
     MockHttpServletRequest request = new MockHttpServletRequest();
     request.addHeader("X-FORWARDED-FOR", "172.1.1.1,172.1.1.2");
     request.setRemoteAddr("172.1.1.3");
     String ip = WebUtils.tryToGetClientIp(request);
     assertThat(ip).isEqualTo("172.1.1.1");
 
     request.removeHeader("X-FORWARDED-FOR");
     ip = WebUtils.tryToGetClientIp(request);
     assertThat(ip).isEqualTo("172.1.1.3");
   }
 }